src/univers/versions.py

```diff
@@ -447,6 +446,33 @@ def build_value(cls, string):
     def __str__(self):
         return f"{self.value[0]}.{self.value[1]}.{self.value[2]}{self.value[3]}"
 
+    def __lt__(self, other):
+        if not isinstance(other, self.__class__):
+            return NotImplemented
+        base_vers1 = self.value[:-1]
+        base_vers2 = other.value[:-1]
+        # Check if versions have the same base, and `one and only one` of them is a pre-release.
+        if base_vers1 == base_vers2 and (
+            self.is_prerelease(self.value) != self.is_prerelease(other.value)
+        ):
+            return self.is_prerelease(self.value)
+        return self.value.__lt__(other.value)
+
+    def __gt__(self, other):
+        if not isinstance(other, self.__class__):
+            return NotImplemented
+        base_vers1 = self.value[:-1]
+        base_vers2 = other.value[:-1]
+        if base_vers1 == base_vers2 and (
+            self.is_prerelease(self.value) != self.is_prerelease(other.value)
+        ):
+            return self.is_prerelease(other.value)
+        return self.value.__gt__(other.value)
+
+    def is_prerelease(self, value):
+        patch = value[-1]
+        return patch.startswith("-beta") or patch.startswith("-alpha")
```

Comment:
This is cleaner and means the same:

```suggestion
        return patch.startswith(("-beta","-alpha",))
```